### PR TITLE
Role component

### PR DESCRIPTION
**Pull request by Tsaikkari**

### CODE REVIEW

File: src/components/Role/index.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { Button } from 'react-bootstrap'
+
+type RoleProps = {
+  text1: string
+  text2: string
+  handleRole: any
```
Comment (Wyfy0107) [inline src/components/Role/index.tsx]: We should type this function also:
`handleRole: () => void`

**Review by Wyfy0107 — APPROVED** [review]
After adding the type, you can merge this pr

**Comment by dilippoudel** [comment]
Good Job, because of less code in this pr, its really interesting to see what did you changes, Otherwise, its really boring

**Review by dilippoudel — APPROVED** [review]

**Comment by Tsaikkari** [comment]
Thanks guys. Merging.

**Pull request merged**